tldw_Server_API/app/api/v1/endpoints/evaluations_unified.py

```
@@ -736,6 +736,96 @@ async def get_rate_limit_status(
 router.include_router(webhooks_router)
 router.include_router(crud_router)
 
+@router.post("", response_model=EvaluationResponse, status_code=status.HTTP_201_CREATED)
+async def create_evaluation_root(
+    eval_request: CreateEvaluationRequest,
+    user_id: str = Depends(verify_api_key),
+    current_user: User = Depends(get_request_user),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)